operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/config/impl/LegacyConfigurationImpl.java

```diff
@@ -74,8 +74,8 @@ public long getCoreFileSizeThreshold() {
 
     @Override
     public int getCorePageSize() {
-        int pageSize = FHIRConfigHelper.getIntProperty("fhirServer/bulkdata/patientExportPageSize", SearchConstants.MAX_PAGE_SIZE);
-        return Math.min(SearchConstants.MAX_PAGE_SIZE, pageSize);
+        int pageSize = FHIRConfigHelper.getIntProperty("fhirServer/bulkdata/patientExportPageSize", FHIRConstants.FHIR_PAGE_SIZE_DEFAULT_MAX);
+        return Math.min(FHIRConstants.FHIR_PAGE_SIZE_DEFAULT_MAX, pageSize);
```

Comment:
I kind of prefer a smaller page size by default 100 versus 1000
Is there a reason we need to boost this to 1000?
I'd rather inline this as 100 versus 1000 by default.

Reply:
specifically related to https://github.com/IBM/FHIR/issues/1868#issuecomment-766913001 
and the type filters are kind of brutal.

Reply:
The previous code used SearchConstants.MAX_PAGE_SIZE, which was 1000. So you like to see the default lowered to 100?

Reply:
Lowered default to 100 just for bulkdata.